feedback/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class GradesConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
+    name = "grades"
```

Comment:
Wydaje mi się że tu powinno być FeedbackConfig
```suggestion
class FeedbackConfig(AppConfig):
    default_auto_field = "django.db.models.BigAutoField"
    name = "feedback"
```